tests_metricflow/query_rendering/test_cumulative_metric_rendering.py

```diff
@@ -35,26 +35,25 @@ def test_cumulative_metric(
     sql_client: SqlClient,
 ) -> None:
     """Tests rendering a basic cumulative metric query."""
-    dataflow_plan = dataflow_plan_builder.build_plan(
-        MetricFlowQuerySpec(
-            metric_specs=(MetricSpec(element_name="trailing_2_months_revenue"),),
-            dimension_specs=(),
-            time_dimension_specs=(
-                TimeDimensionSpec(
-                    element_name="ds",
-                    entity_links=(),
-                    time_granularity=TimeGranularity.DAY,
-                ),
+    query_spec = MetricFlowQuerySpec(
+        metric_specs=(MetricSpec(element_name="trailing_2_months_revenue"),),
+        dimension_specs=(),
+        time_dimension_specs=(
+            TimeDimensionSpec(
+                element_name="ds",
```

Comment:
nit: not really related to the PR, but we don't allow non-metric_time time dims without entity links so it would make this more realistic to add one here.